.github/workflows/build_and_tests.yml

```diff
@@ -18,42 +18,40 @@ jobs:
     - name: Clone
       uses: actions/checkout@v2
     - name: Speculos dependencies
-      run: sudo apt-get update && sudo apt-get install -y qemu-user-static
+      run: sudo apt-get update && sudo apt-get install -y qemu-user-static gcc-arm-linux-gnueabihf libvncserver-dev
     - name: Build & install
       run: |
         pip install --extra-index-url https://test.pypi.org/simple/ -U .[tests,all_backends]
         pip install -U click>=8
-    - name: Gather boilerplate NanoS to run tests against
-      uses: dawidd6/action-download-artifact@v2
-      with:
-        repo: LedgerHQ/app-boilerplate
-        name: boilerplate-app-nanos
-        workflow: ci-workflow.yml
-        path: tests/elfs/
-    - name: Renaming NanoS artifact
-      run: mv tests/elfs/app.elf tests/elfs/nanos.elf
-    - name: Gather boilerplate NanoX to run tests against
-      uses: dawidd6/action-download-artifact@v2
+    - name: Clone Speculos
```

Comment:
To be removed once your speculos Pr is merged I guess?

Reply:
Indeed.